modules/version-update/pages/upgrade-from-community-to-a-subscription-edition.adoc

```diff
@@ -82,9 +85,39 @@ Replace __<platform-setup-install-folder>__ with the folder where you want to in
 Downloading the `platform-setup-sp` tool is optional as it is already packaged in the bundle archive when building your SCA using the `bundle` mode. It is recommended when using a Docker image.
 ====
 
+==== Upgrade a Bonita Runtime with a Tomcat bundle
+
+To upgrade a Bonita Runtime from a Community edition to an Enterprise edition, follow these steps:
+
+. Download and unzip the Bonita Enterprise bundle to `<bonita-enterprise-installation>` folder
+. Copy your database.properties fom your Community installation `setup` folder to `<bonita-enterprise-installation>/setup` folder
+. Run `<bonita-enterprise-installation>/setup/setup(.sh/.bat) pull` to fetch your current Community configuration in `<bonita-enterprise-installation>/setup/platform_conf/current`
```

Comment:
Not usefull to indicate where it is pulled, as the mentioned folder is not used later

```suggestion
. Run `<bonita-enterprise-installation>/setup/setup(.sh/.bat) pull` to fetch your current Community configuration.
```